main.go

```diff
@@ -140,32 +124,27 @@ func timedShuffle(cards []deck.Card) {
 		// Waiting for the channel to emit a value
 		case <-timer.C:
 			// recursively call our shuffle
```

Comment:
This comment doesn't relate to the code underneath it now, so it should be removed or edited.